solutions/USACO open18-team-tic-tac-toe.mdx

```diff
@@ -0,0 +1,167 @@
+---
+id: usaco-831
+title: USACO US Open 2018 - Team Tic Tac Toe
+author: Jesse Choe
+---
+
+[Official Analysis]: http://www.usaco.org/current/data/sol_tttt_bronze_open18.html
```

Comment:
there's already C++ impl...

Reply:
Does it use sets and maps, though?

Reply:
oops mb, you're right. Can we add a quick note after this line that's like "official solution doesn't use sets/maps, here's one that does"?

Reply:
yep ofc. idk if that was directed to @bqi343